code/datums/uplink_items/uplink_general.dm

```diff
@@ -525,6 +525,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/syndie_kit/safecracking
 	cost = 5
 
+/datum/uplink_item/stealthy_tools/handheld_mirror
+	name = "Hand Held Mirror"
+	desc = "Change and color your hair on the go."
```

Comment:
Could describe this item more over here, maybe telling players how small it is and how good it helps with stealth.